terraform-infra-approvals/global.json

```diff
@@ -10,7 +10,8 @@
     {"type": "azurerm_application_insights_web_test"},
     {"type": "azurerm_monitor_action_rule_action_group"},
     {"type": "azurerm_monitor_metric_alert"},
-    {"type": "azurerm_storage_account"}
+    {"type": "azurerm_storage_account"},
+    {"type": "apim_api_operation_policy"}
```

Comment:
why doesn't this use a module?

Reply:
I put this in incorrectly and have corrected/merged : https://github.com/hmcts/sds-jenkins-config/pull/32